index.html

```diff
@@ -0,0 +1,273 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>First project</title>
+        <link type="Image/x-icon" href="./src/img/favicon.ico" rel="icon">
+        <link rel="stylesheet" href="./src/css/reset.css">
+        <link rel="stylesheet" href="./src/css/style.css">
+        <script src="https://kit.fontawesome.com/7e214218fa.js" crossorigin="anonymous"></script>
+        <script src="./src/js/script.js"></script>
+    </head>
+    <body>
+        <div class="container">
+           <header class="header container">
+            <nav class="navbar">
+                <a class="logo navbar__logo">
```

Comment:
WCAG 2.1 requires each <a> anchor link to have a text describing the purpose of the link using either plain text or an <img> with the alt attribute set.